click/core.py

```diff
@@ -1461,7 +1461,7 @@ def __init__(self, param_decls=None, show_default=False,
                  prompt=False, confirmation_prompt=False,
                  hide_input=False, is_flag=None, flag_value=None,
                  multiple=False, count=False, allow_from_autoenv=True,
-                 type=None, help=None, hidden=False, **attrs):
+                 type=None, help=None, hidden=False, show_choices=True, **attrs):
```

Comment:
That doesn't even belong there. It should be in the Option class so that it can be toggled on an per option basis.